autobahn/asyncio/wamp.py

```diff
@@ -172,3 +172,106 @@ def create():
             loop.run_until_complete(protocol._session.leave())
 
         loop.close()
+
+
+# TODO - unify with previous class
+class ApplicationRunnerRawSocket(object):
+    """
```

Comment:
Re: testing with crossbar -   it's already done in example - backend and frontend over raw socket -   backend using unix socket, frontend tcp socket